agent/taskresource/mocks/taskresource_mocks.go

```diff
@@ -52,160 +54,185 @@ func (m *MockTaskResource) EXPECT() *MockTaskResourceMockRecorder {
 }
 
 // ApplyTransition mocks base method
-func (m *MockTaskResource) ApplyTransition(arg0 status0.ResourceStatus) error {
-	m.ctrl.T.Helper()
```

Comment:
what version of gomock are you using? seems like you are reverting the changes [here](https://github.com/aws/amazon-ecs-agent/pull/2115/commits/e0bde51c8934623932faaa1ea6d257a498478991)